packages/core/src/index.js.flow

```diff
@@ -75,7 +75,9 @@ declare export function switchLatest <A> (s: Stream<Stream<A>>): Stream<A>
 
 declare export function merge <A, B> (s1: Stream<A>, s2: Stream<B>): Stream<A | B>
 declare export function merge <A, B> (s1: Stream<A>): (s2: Stream<B>) => Stream<A | B>
-declare export function mergeArray <A> (ss: Array<Stream<A>>): Stream<A>
+
+type MergeArray = <A>(Array<Stream<A>>) => Stream<A>
```

Comment:
Hmmm ... shouldn’t this be:

```typescript
type MergeArray = <A>(ss: Array<Stream<A>>) => Stream<A>
```

And why not `ReadonlyArray<Stream<A>>`?

Reply:
Flow doesn't need the parameter names in function types, only the parameter types.  So, the `ss:` isn't necessary.  Historically, if I remember correctly, we created this flow type declaration file around the time when flow had recently made that change (not requiring parameter names), so we included them for compatibility with folks who might be using previous versions.  At this point, it's safe (it's been years).

It's probably safe to use [Flow's `$ReadonlyArray`](https://flow.org/en/docs/types/arrays/#toc-readonlyarray) at this point, as well.   Thanks 👍 I'll make that change.